app/src/main/java/pm/gnosis/heimdall/ui/two_factor/Connect2FaActivity.kt

```diff
@@ -51,6 +51,9 @@ class ConnectAuthenticatorViewModel @Inject constructor(
             updateState(true) {
                 apply {
                     viewAction = ViewAction.StartActivity(
+                        /*TODO: adjust layout of review transaction ativity
```

Comment:
What exactly needs to be adjusted? Can we create a follow up ticket for that?

Reply:
yes - mostly colors & background. And submit button hast to be part of info panel according to mockups. Follow up ticket follows

Reply:
Ok I just saw that now: the connect flow should not require to use the recovery phrase, so it makes no sense to use the `PairingSubmitActivity` for that. Therefore we should remove `PairingAction.CONNECT`

The design adjustment of `ReviewTransactionActivity` is not specific to KeyCard but a general design adjustment, right?

Reply:
yes, here is the issue: https://app.zenhub.com/workspaces/gnosis-safe-5c1b98250e13551e8aae81eb/issues/gnosis/safe-android/490

Reply:
Why not remove the connect enum in this pr (since it is added in this pr). Also the connect flow already uses the review activity, so no need to have that in the issue. Only the part about adjusting the design is really required.

Reply:
adjusted